Makefile

```diff
@@ -114,8 +121,8 @@ test_go:
 		)
 
 test_e2e:
-	sed -i "s#image:.*$(IMAGE_NAME).*#image: $(DOCKER_IMAGE):$(BUILD_TAG)#" docker-compose.yaml
-	cd plugin/pki/e2e && ginkgo -v
+	sed -i -e "s#image:.*$(IMAGE_NAME).*#image: $(DOCKER_IMAGE):$(BUILD_TAG)#" docker-compose.yaml
```

Comment:
I think we should remove this flag, and keep it as the original

Reply:
-e flag remved from Makefile